### PR TITLE
Prettier inspect

### PR DESCRIPTION
**Pull request by jdoliner**
This adds prettier printing to all of our `inspect` commands.

For example:

```
$ pachctl inspect-pipeline filter
Name: filter
Created: 19 hours ago
State: running
Parallelism: 1
Inputs:
NAME                PARTITION           INCREMENTAL         
data                BLOCK               true                
Transform:
{
  "image": "fruit_stand",
  "cmd": [
    "sh"
  ],
  "stdin": [
    "grep apple  /pfs/data/sales >>/pfs/out/apple",
    "grep banana /pfs/data/sales >>/pfs/out/banana",
    "grep orange /pfs/data/sales >>/pfs/out/orange"
  ]
}

Job Counts:
pulling: 0	running: 0	failure: 0	success: 101	
```

### CODE REVIEW

**Comment by jdoliner** [comment]
Fixes #528 

@derekchiang whenever you have a chance

**Comment by jdoliner** [comment]
This should have a test job counts.

**Comment by jdoliner** [comment]
There's a bug with the InspectCommit template that I discovered while demoing. >.<

File: src/client/pps/pps.proto [hunk 1]
```diff
@@ -92,6 +92,7 @@ message PipelineInfo {
   google.protobuf.Timestamp created_at = 6;
   PipelineState state = 7;
   string recent_error = 8;
+  map<int32, int32> job_counts = 9;
```
Comment (derekchiang) [inline src/client/pps/pps.proto]: State to number?  Might want a comment.

**Comment by derekchiang** [comment]
Prettier == better

File: src/client/pps/pps.proto (hunk 1 above)
Comment (jdoliner) [inline src/client/pps/pps.proto]: Good point, ridiculously enough proto won't let me use states as map keys... even though they compile to int32s. I guess maybe that wouldn't work in other languages?

**Pull request merged**